test/bdd/steps/network.js

```diff
@@ -510,6 +510,23 @@ Then(/^the last two datasets should have the same hashes$/, async function () {
         .to.be.equal(this.state.secondLastImport.data.dataset_id);
 });
 
+Given(/^(DC|DV|DV2) waits for public key request$/, { timeout: 1200000 }, async function (targetNode) {
```

Comment:
Is the 20 minute really necessary? Especially because we don't have a rejection case, the only way to see if this test failed is to wait 20 minutes. I think we should either add a failure case or reduce the timeout for the step